metadata.rb

```diff
@@ -3,9 +3,15 @@
 name             'netuitive'
 maintainer       'Ben Abrams'
 maintainer_email 'me@benabrams.it'
-license          'All rights reserved'
+license          'MIT'
```

Comment:
@majormoses we typically use Apache 2.0. Would that be ok with you here?

Reply:
Only thing is that it's already really licensed under MIT, we could dual license it as both.

Reply:
@majormoses oh, my mistake! No problem, then. Thanks for clarifying!